### PR TITLE
Fix for the nextcloud 21 migration

### PR DESCRIPTION
**Pull request by stephdl**
- verify the SCL nextcloud DB is true before to  use the socket in nextcloud for the new database
- verify that the SCL DB is not blank before to remove the mariadb55 nextcloud database 
- disable accessibility to save PHP ressources (beware it is a mandatory plugin)

https://github.com/NethServer/dev/issues/6506

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf [hunk 1]
```diff
@@ -11,10 +11,16 @@ function OCC
...
     echo $1
+    OCC "app:enable accessibility"
     OCC "maintenance:mode --off"
     exit 1
 }
 
+databaseTest () {
+    # if the database is not initialized then databaseSCL == 0 else > 0
+    databaseSCL=`mysql --socket=/run/rh-mariadb105-mariadb/nextcloud-mysql.sock -e "select count(*) from information_schema.tables where table_type = 'BASE TABLE' and table_schema = 'nextcloud'" | tail -n1`
+}
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-nextcloud-conf]: Please do not set a global variable, just return an error code.
Something like:
```
databaseTest () {
   cmd
   return $?
}

if databaseTest; then
...
fi
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf [hunk 2]
```diff
@@ -11,10 +11,16 @@ function OCC
...
 exitOnError () {
     echo $1
+    OCC "app:enable accessibility"
     OCC "maintenance:mode --off"
     exit 1
 }
 
+databaseTest () {
+    # if the database is not initialized then databaseSCL == 0 else > 0
+    databaseSCL=`mysql --socket=/run/rh-mariadb105-mariadb/nextcloud-mysql.sock -e "select count(*) from information_schema.tables where table_type = 'BASE TABLE' and table_schema = 'nextcloud'" | tail -n1`
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-nextcloud-conf]: You could also avoid the `tail -n1` invocation by using:
```
 mysql --socket=/run/rh-mariadb105-mariadb/nextcloud-mysql.sock -BN -e "select count(*) from information_schema.tables where table_type = 'BASE TABLE' and table_schema = 'nextcloud'"
```

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf (hunk 1 above)
Comment (stephdl) [reply root/etc/e-smith/events/actions/nethserver-nextcloud-conf]: done

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf (hunk 2 above)
Comment (stephdl) [reply root/etc/e-smith/events/actions/nethserver-nextcloud-conf]: great nice thank

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**